xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -567,7 +565,7 @@ func newConfigFromContents(data []byte) (*Config, error) {
 	}
 	node.UserAgentName = gRPCUserAgentName
 	node.UserAgentVersionType = &v3corepb.Node_UserAgentVersion{UserAgentVersion: grpc.Version}
-	node.ClientFeatures = append(node.ClientFeatures, clientFeatureNoOverprovisioning, clientFeatureResourceWrapper)
+	node.ClientFeatures = AppendIfNotPresent(node.ClientFeatures, clientFeatureNoOverprovisioning, clientFeatureResourceWrapper)
```

Comment:
Thanks for making the change. I think with the last commit, this can also be reverted